rosflight/CMakeLists.txt

```diff
@@ -32,6 +32,15 @@ if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/include/rosflight/mavlink/v1.0/.git")
   )
 endif()
 
+execute_process(COMMAND git describe --tags --abbrev=8 --always --dirty --long WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} OUTPUT_VARIABLE GIT_VERSION_STRING RESULT_VARIABLE GIT_RESULT)
+if(GIT_RESULT EQUAL 0)
+	string(REGEX REPLACE "\n$" "" GIT_VERSION_STRING "${GIT_VERSION_STRING}")
```

Comment:
why is this here again?  removing newline characters?

Reply:
Yes, I've added a comment.